packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.html

```diff
@@ -698,10 +878,147 @@
+                                        >
+                                        </strong>
+                                        <label class="d-block">
+                                            <div
+                                                class="input-group d-inline-block align-middle mr-2"
+                                            >
+                                                <input
+                                                    type="checkbox"
+                                                    data-ng-model="offer.options.gtr_10ho.selected"
+                                                    data-ng-change="$ctrl.updateOfferDisplayedPrice(offer.options.gtr_10ho, offer, offer.options.gtr_10ho.name);
+                                                                    $ctrl.constructor.updateSelectedGtrOption(offer.options.gtr_10ho, offer, offer.options.gtr_10ho.name)"
+                                                />
+                                            </div>
+                                            <small class="d-block align-middle">
+                                                <span
+                                                    data-translate="telecom_pack_migration_gtr_add"
+                                                ></span>
+                                                <span
+                                                    data-ng-bind-html="offer.options.gtr_10ho.optionalPrice.text | tucFormatPrice:{ addBrackets : true }"
+                                                ></span>
+                                            </small>
+                                        </label>
+                                    </div>
+                                    <div data-ng-if="offer.options.gtr_4ho">
+                                        <strong
+                                            class="d-block"
+                                            data-translate="telecom_pack_migration_gtr_4ho_available"
+                                        >
+                                        </strong>
+                                        <label class="d-block">
+                                            <div
+                                                class="input-group d-inline-block align-middle mr-2"
+                                            >
+                                                <input
+                                                    type="checkbox"
+                                                    data-ng-model="offer.options.gtr_4ho.selected"
+                                                    data-ng-change="$ctrl.updateOfferDisplayedPrice(offer.options.gtr_4ho, offer, offer.options.gtr_4ho.name);
+                                                                    $ctrl.constructor.updateSelectedGtrOption(offer.options.gtr_4ho, offer, offer.options.gtr_4ho.name)"
```

Comment:
**suggestion:** Use a common function and call the 3 functions inside the common function.

Reply:
done